.github/workflows/lte-integ-test-bazel.yml

```diff
@@ -74,51 +75,28 @@ jobs:
       - name: Build all services with bazel
         run: |
           cd lte/gateway
-          vagrant ssh -c 'cd ~/magma; bazel/scripts/remote_cache_bazelrc_setup.sh "${{ env.CACHE_KEY }}" "${{ env.REMOTE_DOWNLOAD_OPTIMIZATION }}" "${{ secrets.BAZEL_REMOTE_PASSWORD }}";' magma
+          vagrant ssh -c 'cd ~/magma; bazel/scripts/remote_cache_bazelrc_setup.sh "${{ env.CACHE_KEY_MAGMA_DEV }}" "${{ env.REMOTE_DOWNLOAD_OPTIMIZATION }}" "${{ secrets.BAZEL_REMOTE_PASSWORD }}";' magma
           vagrant ssh -c 'sudo sed -i "s@#precedence ::ffff:0:0/96  100@precedence ::ffff:0:0/96  100@" /etc/gai.conf;' magma
           vagrant ssh -c 'cd ~/magma; bazel build --profile=bazel_profile_lte_integ_tests `bazel query "kind(.*_binary, //orc8r/... union //lte/... union //feg/...)"`;' magma
           vagrant ssh -c 'sudo sed -i "s@precedence ::ffff:0:0/96  100@#precedence ::ffff:0:0/96  100@" /etc/gai.conf;' magma
-      - name: Run the integ test
+      - name: Setup magma test VM and traffic server VM
         run: |
           cd lte/gateway
           export MAGMA_DEV_CPUS=3
           export MAGMA_DEV_MEMORY_MB=9216
           fab bazel_integ_test_post_build
+      - name: Run the integ tests
+        run: |
+          cd lte/gateway
+          vagrant ssh -c 'cd ~/magma; bazel/scripts/remote_cache_bazelrc_setup.sh "${{ env.CACHE_KEY_MAGMA_TEST }}" "${{ env.REMOTE_DOWNLOAD_OPTIMIZATION }}" "${{ secrets.BAZEL_REMOTE_PASSWORD }}";' magma
+          # The ssh option '-ttt' is forcing tty allocation, even if ssh has no local tty.
+          vagrant ssh -c 'cd ~/magma; bash -c "bazel/scripts/run_integ_tests.sh"' magma_test -- -ttt
       - name: Publish bazel profile
         uses: actions/upload-artifact@3cea5372237819ed00197afe530f5a7ea3e805c8 # pin@v3
         if: always()
         with:
           name: Bazel profile lte integ tests
           path: bazel_profile_lte_integ_tests
-      - name: Get test results
```

Comment:
why are these not relevant for the bazel workflow?

Reply:
This will be added back in once we add the XML generation (this is done on a different issue), which includes the sudo tests.